manifests/backup/backup-gcs.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: pingcap.com/v1alpha1
+kind: Backup
+metadata:
+  name: demo1-backup-gcs
+  namespace: test1
+spec:
+  gcs:
+    projectNumber: your-project-id
```

Comment:
```suggestion
    projectNumber: <your-project-id>
```

Reply:
and all other vars should be changed to `<your-xxxx>`?

Reply:
I think to replace this var with` <your-xxx>` is enough